sklego/pandas_utils.py

```diff
@@ -11,45 +11,154 @@
 from scipy.ndimage.interpolation import shift
 
 
-def log_step(func=None, *, level=logging.INFO):
+def get_shape_delta(old_shape, new_shape):
```

Comment:
Since this function isn't public, better translate it to `_get_shape_delta`.

Reply:
Done